src/main/resources/public/src/views/DataRequest.vue

```diff
@@ -84,7 +96,7 @@
         },
         computed: {
             disabled: function(){
-                return (this.organizationSelected == false || this.dataAccessRequest.email == '' || this.modelSelected == '')
+                return (this.organizationSelected === false || this.dataAccessRequest.email === '' || this.modelSelected === '')
             }
```

Comment:
If you initialize modelSelected to null value, you need to add it in the computed disabled condition otherwise users will be able to submit forms without selecting models on first screen opening.